semana12/planner/src/actions/tasks.test.js

```diff
@@ -0,0 +1,50 @@
+import  { setTasks, getTasks, createTask } from "./tasks"
+import axios from "axios"
+
+describe ("Tasks Actions", () => {
+    test ("Set Tasks", () => {
```

Comment:
Tente ser mais clara nas strings de primeiro parâmetro dos testes. Ela deve descrever o que você está querendo testar. Se vc colocar apenas "Set Tasks", seu código não fica muito claro. Seria mais legal se vc colocasse uma descrição como: "should test the returns of the setTask action".